testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7189,6 +7189,43 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
         "INTEGER ARRAY NOT NULL");
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6223">[CALCITE-6223]
+   * Add SAFE_NEGATE function (enabled in BigQuery library)</a>.
+   */
+  @Test void testMapContainsKeyFunc() {
+    final SqlOperatorFixture f0 = Fixtures.forOperators(true);
```

Comment:
I don't think we normally instantiate the test fixture this way. Just use `f0 = fixture().setFor(...)` as normal.

Reply:
sorry，fix